### ISSUE TITLE
Roads level 0 - calculate swamp road (bridge) positions.

### ISSUE THREAD

**Issue by karlthepagan** [ScreepsOCS/screeps.behaviour-action-pattern#967]
Road building ideas. Room bootstrapping is greatly impacted by swamps. First road priority should be critical swamp crossings. For build speed find the smallest transit.

Probably need a good scoring function for this one.

**Comment by DasBrain** [comment]
Here my 2 cents:

- If a creep has fatigue, let it tick more
- If the terrain is swamp, require less ticks?

**Comment by karlthepagan** [comment]
@DasBrain if you are talking about movement rules, these are covered in traveler.js

This ticket is for iterating on automatic road building ideas.

**Comment by DasBrain** [comment]
Yeah. Currently we increase a counter for each field when we traverse it ([room.js:1172](https://github.com/ScreepsOCS/screeps.behaviour-action-pattern/blob/dev/room.js#L1172)), so my idea was simply to increase it by more points (e.g. 2) if the creep has fatigue or stands on swamp. One of them. Or both.

**Comment by karlthepagan** [comment]
Ok, I see so you'd favor continuing to use the existing road traces.
